python/vineyard/contrib/ml/pytorch.py

```diff
@@ -83,12 +105,12 @@ def torch_tensor_resolver(obj):
     data = torch.from_numpy(np.frombuffer(memoryview(obj.member('buffer_data_')), dtype=data_type).reshape(data_shape))
     label = torch.from_numpy(
         np.frombuffer(memoryview(obj.member('buffer_label_')), dtype=label_type).reshape(label_shape))
-    dataset = torch.utils.data.TensorDataset(data, label)
-    return dataset
+    return torch.utils.data.TensorDataset(data, label)
 
 
 def torch_dataframe_resolver(obj, resolver):
     meta = obj.meta
+    target = from_json(meta['target'])
```

Comment:
Would you mind change the keyword 'target' into 'label' coordinated to the case of xgboost? And please set the keyword as kwargs to client.get, because you can't guarantee every dataframe builder will put the 'target' meta in the vineyard::dataframe it builds.